langchain/llms/gpt4all.py

```diff
@@ -28,6 +28,8 @@ class GPT4All(LLM):
     model: str
     """Path to the pre-trained GPT4All model file."""
 
+    backend: str = Field("llama", alias="backend")
```

Comment:
out of curiosity, what's value of specifying alias in this case?

Reply:
Without it is doesn't work. When creating an instance it will complain that it doesn't see backend.
`llm = GPT4All(model=local_path, backend='gptj', callbacks=callbacks, verbose=True, temp=0.5)`